src/circuit.rs

```diff
@@ -92,6 +92,91 @@ pub struct Cell {
     column: Column<Any>,
 }
 
+/// An assigned cell.
+#[derive(Clone, Copy, Debug)]
+pub struct AssignedCell<F: Field, T: Copy + Into<F>> {
```

Comment:
`Copy` is overly strict, you could use `Clone` instead. (Which will effectively act as a copy for `Copy` types.)